docs/VnVPlan/VnVPlan.tex

```diff
@@ -231,7 +219,7 @@ \subsection{Design Verification Plan}
 
 \subsection{Verification and Validation Plan Verification Plan}
 
-to ensure the quality and completeness of the verification and
+To ensure the quality and completeness of the verification and
```

Comment:
this is just because I realized that the first letter of the sentence isn't capitalized when it should be